### ISSUE TITLE
Dollar Sign ($) String Interpolation is not Supported in .NET 3.5

### ISSUE THREAD

**Issue by tjhalva** [twilio/twilio-csharp#406]
Effort has been put forth to support .net 3.5 but the $ string object concatenation feature was added in C# 6.0 and breaks compilation when compiling on any framework less than 4.6. Suggestion is to utilize normal concatenation in this case.

**Version: 5.9.2**

### Code Snippet
```csharp
// Line 46 of EventStreamScope.cs
return $"{Scope}?{queryString}";

// Must be changed to something like below in order to compile without C# 6.0
return Scope + "?" + queryString;
```

Changes have been made to the jwt namespace and a pull request generated. I am unsure whether there is an intent to support compilation on older C# versions. I had to back port to .net 4.0 so I had the changes and I'm presenting them in case there is interest.

https://github.com/twilio/twilio-csharp/pull/407

**Comment by efossier** [comment]
@tjhalva I've reviewed your PR and we'll get this out next release, thanks for fixing it yourself.

**Comment by childish-sambino** [comment]
Fixed by #407
